alembic/versions/a3d71270256c_adds_panel_application_fields.py

```diff
@@ -0,0 +1,79 @@
+"""Adds panel application fields
```

Comment:
Also, I'm going to remove the terms "additional" and "more" because they're redundant. Trying to make it concise and descriptive is actually quite difficult ☹️